src/system/WatchableEventManagerSelect.cpp

```diff
@@ -356,8 +356,13 @@ void WatchableEventManager::HandleEvents()
         watchable->SetPendingIO(
             SocketEventsFromFDs(watchable->GetFD(), mSelected.mReadSet, mSelected.mWriteSet, mSelected.mErrorSet));
     }
-    for (WatchableSocket * watchable = mAttachedSockets; watchable != nullptr; watchable = watchable->mAttachedNext)
+
+    WatchableSocket * watchableSocket = mAttachedSockets;
```

Comment:
naming wise, I find it somewhat hard to differentiate bewtween 'watchable' and 'watchableSocket' (they sound the same). How about `nextWatchableSocket` and `currentWatchable` or similar?

Reply:
Acked